django/contrib/gis/db/backends/postgis/const.py

```diff
@@ -7,6 +7,8 @@
 # Lookup to convert pixel type values from PostGIS to GDAL
 POSTGIS_TO_GDAL = [1, 1, 1, 3, 1, 3, 2, 5, 4, None, 6, 7, None, None]
 
+# see https://trac.osgeo.org/postgis/wiki/WKTRaster/RFC/RFC1_V0SerialFormat
```

Comment:
Please move this ref below, to the place when we actually use it (for defining flags).